rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/HostsFileBasedInfrastructureManager.java

```diff
@@ -54,6 +49,11 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import org.objectweb.proactive.core.node.Node;
+import org.ow2.proactive.resourcemanager.exception.RMException;
+import org.ow2.proactive.resourcemanager.nodesource.common.Configurable;
+import org.ow2.proactive.utils.FileToBytesConverter;
+
 
 /** Abstract infrastructure Manager implementation based on hosts list file. */
 public abstract class HostsFileBasedInfrastructureManager extends InfrastructureManager {
```

Comment:
unrelated to your change, but for clarity, it would be better to rename the map freeHosts to hostsToDeploy